packages/console/src/components/Steps/components/StepItem/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { type AdminConsoleKey } from '@logto/phrases';
+import { type ReactNode } from 'react';
+
+import DynamicT from '@/components/DynamicT';
+
+import Status, { type StepStatus } from './Status';
+import * as styles from './index.module.scss';
+
+export type StepItemProps = {
+  step: number;
+  currentStep: number;
+  status: StepStatus;
+  title: AdminConsoleKey;
+  content?: ReactNode;
+  isContentVisibleOnFinished?: boolean;
```

Comment:
Is this an item-level prop?  Could we have some items visible on the finished status and some are not?

Reply:
This is a design requirement.
<img width="562" alt="image" src="https://github.com/logto-io/logto/assets/10806653/94415f5b-8e98-41a4-a4ef-be1c5827d0c8">